### DISCUSSION TITLE
missing ddl files when installing games

### DISCUSSION THREAD

**Discussion by theunofficialboss** [legendary-gl/legendary#508] [Q&A]
Hello, I'm pretty new to Legendary and I have a problem launching Amnesia: A Machine for Pigs. I'm using Windows 10.

The game is installed, but when running the game has an error saying that MSVCP100.dll is missing.

I tried downloading that dll file from online and putting into the game directory but it still wouldn't launch.

I just updated Microsoft Visual C++, but it might be a directx issue. Please help!

**Comment by CommandMC** [comment]
Make sure you have both the X86 and X64 versions of VCRuntime installed (the EGL installs them in the background, Legendary does not)
https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022

**Reply by theunofficialboss** [reply]
Cheers. I'll have a try.

**Reply by theunofficialboss** [reply]
I have installed both, but the game still reports that the dll file is missing. Should I uninstall and reinstall the game?

**Reply by theunofficialboss** [reply]
I tried a reinstall. The dll file still seems missing.
